express-get-json/index.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+app.listen(port, () => {
```

Comment:
We usually call app.listen at the very end of the file. I'm not sure if it matters, but you should probably follow that convention.